.github/CODEOWNERS

```diff
@@ -0,0 +1,15 @@
+# Gabriel as default owner
+*   @gabalafou
+
```

Comment:
Should also add /examples/ - Noa as default

Reply:
Added!